diesel_codegen_syntex/src/update.rs

```diff
@@ -53,13 +51,14 @@ fn table_name(cx: &mut ExtCtxt, meta_item: &MetaItem) -> Result<InternedString,
     }
 }
 
+#[allow(unused_imports)] // quote_tokens! generates warnings
 fn boolean_option(cx: &mut ExtCtxt, meta_items: &[P<MetaItem>], option_name: &str)
-    -> Result<Option<bool>, ()>
+    -> Result<Vec<TokenTree>, ()>
```

Comment:
Why did you change this fn to return a token tree vec? IIUC this is given the same context as `changeset_impl` so you could just return a bool here and match on it to create the quoted version in `changeset_impl` below.